src/containers/Staking/components/Validators/components/DelegationModal/Success.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  ArrowCircleRightRounded as ArrowCircleRightRoundedIcon,
+  InfoRounded as InfoRoundedIcon,
+  OpenInNewRounded as OpenInNewRoundedIcon
+} from '@mui/icons-material'
+import { Box, Typography, Divider, Stack, Button } from '@mui/material'
+import { ModalProps } from 'store/validator'
+
+import getMiddleEllipsis from '../../../../../../utils/get_middle_ellipsis'
+import SuccessIcon from '../../../../../../assets/vectors/success.svg'
+import { ModalContainer, CancelRoundedIcon } from './styles'
+
+type SuccessProps = {
+  modalProps: ModalProps
+  handleModal: (modalProps: ModalProps) => void
+}
+
+const Success: React.FC<SuccessProps> = ({ modalProps, handleModal }) => {
+  const { validator, amount } = modalProps
+
+  const handleClose = () => {
+    handleModal({ open: false, validator: null, amount: null, status: null })
+  }
+
+  return (
+    validator && (
+      <ModalContainer sx={{ padding: '4rem' }}>
+        <img src={SuccessIcon} alt="success-icon" />
+        <CancelRoundedIcon onClick={handleClose} />
+        <Box display="flex" flexDirection="column" alignItems="center" gap={1}>
+          <Typography variant="h4" fontWeight={900} letterSpacing={2}>
+            Success!
+          </Typography>
+          <Typography variant="subtitle1" color="text.secondary">
+            Transaction was successfully executed
```

Comment:
Maybe an exclamation mark on the end of line 35.

Reply:
fixed